src/js/experiments/popupCta.js

```diff
@@ -43,26 +58,20 @@ function getPopupData() {
  * and determine if we can show the popup.
  * */
 function canShowPopup() {
-  let now = new Date();
-  let stillTheSameDay = false;
   const popupData = getPopupData();
-  if (popupData.lastSeen !== null) {
-    const popupLastSeen = new Date(popupData.lastSeen);
-    stillTheSameDay =
-      now.getFullYear() === popupLastSeen.getFullYear() &&
-      now.getMonth() === popupLastSeen.getMonth() &&
-      now.getDate() === popupLastSeen.getDate();
```

Comment:
This block ( was extracted into the function "isStillTheSameDay()" as it's now being called twice. I passed in the local storage value rather than having `isStillTheSameDay` be responsible for running `getPopupData` since the local storage data was available in the outer scope both places that it is invoked.